packages/ui-library/src/components/transaction-receipt/index.tsx

```diff
@@ -619,26 +620,78 @@ const EarnWithdrawTransactionReceipt = ({ transaction }: { transaction: EarnWith
   const { spacing } = useTheme();
   const intl = useIntl();
 
+  return (
+    <>
+      <StyledSectionContent>
+        <Typography variant="labelRegular">
+          <FormattedMessage description="TransactionReceipt-transactionEarnWithdraw" defaultMessage="Withdrew" />
```

Comment:
wait are we parsing all sending of delayed withdraw and normal withdraws with the same receipt? Or does this only apply if you also withdrew some token through market?

Reply:
Yes, they are all the same event, with a different withdrawType prop. I've only added the withdraw Type in the delayed receipt, becuase it's the one that doesn't put money immediately.
Should we have three different receipts?